empress/plugin_setup.py

```diff
@@ -137,13 +137,19 @@
         'tree': Phylogeny[Rooted]
     },
     parameters={
-        'feature_metadata': Metadata
+        'feature_metadata': Metadata,
+        'shear_to_feature_metadata': Bool
     },
     input_descriptions={
         'tree': TREE_DESC
     },
     parameter_descriptions={
-        'feature_metadata': FM_DESC
+        'feature_metadata': FM_DESC,
+        'shear_to_feature_metadata': (
+            'Determines whether or not to shear the tree to only the tips '
+            'that are present as features in the feature metadata. By '
```

Comment:
```suggestion
            'Determines whether or not to shear the tree to only the tips '
            'that are present as features in the feature metadata. Internal '
            'nodes present in the feature metadata are ignored during '
            'shearing, and can potentially be filtered out if none of their '
            'child tips are present in the feature metadata. By '
```

feel free to reword this as you see fit. just wanted to make this abundantly clear here so people don't get confused.